src/Library/demos/Welcome/main.py

```diff
@@ -0,0 +1,36 @@
+import gi
+
+gi.require_version("Gtk", "4.0")
+gi.require_version("Adw", "1")
+from gi.repository import Gtk, Adw
+import workbench
+
+
+def greet(_widget):
+    dialog = Adw.MessageDialog(
+        body="Hello World!",
+        transient_for=workbench.window
+    )
+
+    dialog.add_response("ok", "Ok")
+    dialog.connect("response", dialog_response)
+    dialog.present()
+
+
+def dialog_response(dialog: Adw.MessageDialog, response: str):
+    print(response)
+    dialog.close()
```

Comment:
Python lambdas can only have a single statement, the language discourages lambdas and encourages using named functions.

Reply:
We COULD just use a Lambda here and just close without printing a language. I think by using parens it would even be possible to run both statements, but it's really not something you are supposed to do.